### ISSUE TITLE
bad domain in WDSL causes exit

### ISSUE THREAD

**Issue by nomcode** [mvantellingen/python-zeep#878]
zeep 3.1.0
python 3.5.2

Not sure if this is an error or problem with zeep but more likely requests.
However should zeep (or requests) be trying to resolve the domain name found in the XML ?
Certainly should it be falling out of the program if an error is found?
using "python -mzeep WDSL-URL" doesn't cause a problem.
Prog::

import zeep
wsdl = 'http://www.hennvr-ddns.com/webservice/GetDeviceInterface?wsdl'
client = zeep.Client(wsdl=wsdl)
test = client.service.getMacInfo('','')
print(test)

Error::
requests.exceptions.ConnectionError: HTTPConnectionPool(host='hennvr_ddns', port=80): Max retries exceeded with url: /webservice/GetDeviceInterface (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x04016E70>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed',))

It's true, the domain name is invalid in the XML. But that shouldn't cause a problem?
Is there a way to avoid resolving all the domains found in the XML ?
Not sure if bug or if there is a config setting that can avoid this issue.
Have submitted a bug to try to get server side fixed.

XML result with the issue from server::
  <wsdl:service name="GetDeviceInterfaceImplService">
    <wsdl:port binding="tns:GetDeviceInterfaceImplServiceSoapBinding" name="GetDeviceInterfaceImplPort">
      <soap:address location="http://hennvr_ddns/webservice/GetDeviceInterface"/>
    </wsdl:port>
  </wsdl:service>

**Comment by mvantellingen** [comment]
I'm closing this issue due to the low quality of the initial report. You are welcome to create a new issue but please make sure it contains all the relevant information and that it is properly formatted.

Also please remember that this project is opensource and completely unpaid work.

**Comment by nomcode** [comment]
HI,

Honestly I thought it was well formatted. The issue, versions, the program the error response, the XML result.
Its' got a link to the WDSL interface and the 5 lines of a python program to recreate the problem.
Not sure what else is relevant?

I'm not sure what else I can put into the bug report but you say it's "low quality" ?
I'm not an experienced programmer so if I'm missing something others would expect to be in there I'm sorry but I lack the experience to know what that might be, well a month isn't experienced, but pointers most welcome thanks.
I read the link to the read before reporting a bug, and I thought I followed it as above. I didn't try the example demo programs because my error isn't allowing me to get to the point where I call a method name, it's before that when parsing out the methods that the error occurs.

I do appreciate that it's unpaid work, I didnt demand anything, I posted a bug report which I'd imagine other people would have the same issue if the WDSL domain was incorrect for them too so seemed relevant?
